frontend/src/component/personalDashboard/ConnectSDK.tsx

```diff
@@ -48,6 +64,30 @@ export const CreateFlag: FC<{ project: string }> = ({ project }) => {
     );
 };
 
+export const ExistingFlag: FC<{ project: string }> = ({ project }) => {
+    return (
+        <ActionBox>
+            <TitleContainer>
+                <MainCircleContainer>✓</MainCircleContainer>
+                Create a feature flag
+            </TitleContainer>
+            <SuccessContainer>
+                <Typography fontWeight='bold' variant='body2'>
+                    You have created your first flag
+                </Typography>
+                <Typography variant='body2'>
+                    Go to project to customize the flag further
+                </Typography>
+            </SuccessContainer>
+            <div>
+                <Button href={`projects/${project}`} variant='contained'>
+                    Go to project
+                </Button>
```

Comment:
Can we make this a Link instead? Sure, we can style it as a button, but it *is* a link, so let's make it one 💁🏼

Reply:
Yup, we'll do it in one go as a separate thing